pkgs/tools/misc/routino/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchsvn, perl, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  name = "Routino-${version}";
```

Comment:
Lowercase routino? Possible typo? If not sorry for the noise.

Reply:
https://nixos.org/nixpkgs/manual/#sec-package-naming :
> Don’t use uppercase letters in the name attribute — e.g., "mplayer-1.0rc2" instead of "MPlayer-1.0rc2".